app/views/govuk_component/related_items.raw.html.erb

```diff
@@ -0,0 +1,34 @@
+<%
+  sections ||= []
+%>
+<aside class="govuk-related-items" role="complementary">
+  <% sections.each do |section| %>
+    <% section_title = section[:title] || "Elsewhere on GOV.UK" %>
```

Comment:
Should this be a default in the component, or up to the code calling the component to specify?